src/components/Kv/KvAccordionItem.vue

```diff
@@ -1,30 +1,26 @@
 <template>
 	<div
-		class="kv-accordion"
-		:class="{
-			'kv-accordion--open' : isOpen,
-		}"
+		class="tw-border-b tw-border-tertiary tw-relative last:tw-border-b-0"
 	>
 		<button
-			class="kv-accordion__header-button"
+			class="tw-w-full tw-flex tw-justify-between tw-items-center tw-py-1.5 tw-px-0
+				tw-text-left disabled:tw-cursor-not-allowed disabled:tw-opacity-low"
```

Comment:
Sorry, just noticed one other thing. The old accordion had a hover color of blue. I think we need to add
`hover:tw-text-action-highlight`

Reply:
Good.
Hover and focus options were added